helm/azure-workload-identity-webhook/values.yaml

```diff
@@ -17,7 +17,7 @@ resources:
     memory: 30Mi
```

Comment:
Sorry I meant the limit 🙈 (brain still waking up).
You can revert the request to `30Mi`.